dumb_pypi/main.py

```diff
@@ -150,7 +150,7 @@ def create(
             upload_timestamp: Optional[int] = None,
             uploaded_by: Optional[str] = None,
     ) -> 'Package':
-        if not re.match('[a-zA-Z0-9_\-\.]+$', filename) or '..' in filename:
+        if not re.match('[a-zA-Z0-9_\-\.\+]+$', filename) or '..' in filename:
```

Comment:
while you're at it (not your fault, existing code -- sorry!) can you add a `r` prefix to this regex -- (it'll become a syntax error in python3.8 -- heh)